src/main/java/ai/pluggy/client/request/Options.java

```diff
@@ -0,0 +1,10 @@
+package ai.pluggy.client.request;
+
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+public class Options {
```

Comment:
I think `Options` is a too generic name, but specific for connect token. I'd rename it to `ConnectTokenOptions`

Reply:
thanks for the suggestion